models/user.go

```diff
@@ -23,6 +23,7 @@ type User struct {
 	Name               string    `db:"name" json:"name"`
 	Password           string    `db:"password" json:"password"`
 	StripeCustomerID   string    `db:"stripe_customer_id" json:"stripe_customer_id"`
+	ExpressAccountID   string    `db:"express_account_id" json:"express_account_id"`
```

Comment:
Can we change this to StripeConnectAccountID? Express is the type of connect account, and we may change types in the future.